pkg/mcs/exportspec.go

```diff
@@ -156,13 +181,20 @@ func (es *ExportSpec) IsPreferredOver(another *ExportSpec) bool {
 // Retruns true when the specifications are compatible, otherwise returns
 // false and the name of the conflicting field.
 // @todo do we want to collect a map of the conflicting Global Properties?
-func (es *ExportSpec) IsCompatibleWith(another *ExportSpec) (bool, string) {
+func (es *ExportSpec) IsCompatibleWith(another *ExportSpec) *CompatibilityError {
+	if es.Name != another.Name {
+		return &CompatibilityError{another.ClusterID, NameField}
```

Comment:
Please initialize struct fields by name and not implicitly (e.g., we have two strings in this case, what if you accidently reverse them? It won't be caught by the compiler... See https://github.com/uber-go/guide/blob/master/style.md#use-field-names-to-initialize-structs)